build-langs

```diff
@@ -12,9 +12,9 @@ declare -A urls=(
     ["Brainfuck"]="//github.com/code-golf/code-golf/tree/master/langs/brainfuck/"
     ["C"]="//bellard.org/tcc/"
     ["Go"]="//golang.org"
-    ["Fortran"]="https://gcc.gnu.org/fortran/"
+    ["Fortran"]="//gcc.gnu.org/fortran/"
     ["Haskell"]="//www.haskell.org/ghc/"
-    ["J"]="http://jsoftware.com"
+    ["J"]="//jsoftware.com"
```

Comment:
this wont work will it? j doesn't support https does it?

Reply:
It does now.

Reply:
oh awesome

Reply:
looks like it needs the www. my end though, or i get cert warnings